.github/actions/setup/action.yml

```diff
@@ -100,6 +106,12 @@ runs:
         create_credentials_file: true
         workload_identity_provider: ${{ inputs.gcp-provider }}
         service_account: ${{ inputs.gcp-service-account }}
+    - name: Configure AWS Credentials
+      if: inputs.aws-role != ''
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: ${{ inputs.aws-role }}
+        aws-region: us-east-1
```

Comment:
question (non-blocking): For my own understanding, why would we want to skip this if aws-role is empty? Is this conditional for the case where it is run from dependabot / external contributors?

Reply:
We use the setup action for everything, including jobs that don't need AWS access and runs from external contributors that don't have secrets access at all, so this lets us skip trying to log in to AWS in both of those cases.